opensrp-giz-malawi/src/main/java/org/smartregister/giz/recievers/GizReprocessSyncStatusReceiver.java

```diff
@@ -0,0 +1,28 @@
+package org.smartregister.giz.recievers;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+import org.smartregister.giz.job.GIZClientReprocessJob;
+
+import timber.log.Timber;
+
+import static org.smartregister.receiver.SyncStatusBroadcastReceiver.EXTRA_COMPLETE_STATUS;
+
+public class GizReprocessSyncStatusReceiver extends BroadcastReceiver {
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        Bundle data = intent.getExtras();
+        if (data != null) {
+            boolean isComplete = data.getBoolean(EXTRA_COMPLETE_STATUS, false);
+            if (isComplete) {
+                GIZClientReprocessJob.scheduleJobImmediately(GIZClientReprocessJob.TAG);
+                Timber.d("Sync complete");
+            }
+        }
+
+        Timber.d("broadcast Recieved");
```

Comment:
I think the log is too vague and it would be easier to understand if it was a bit more specific on the broadcast that was received